configurations/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
```

Comment:
@lascenify I used the configuration file that Typescript generates by default. We can iterate over this structure as the project evolves.